cmd/sonobuoy/app/run.go

```diff
@@ -82,11 +83,12 @@ func submitSonobuoyRun(f *genFlags) func(cmd *cobra.Command, args []string) {
 			os.Exit(1)
 		}
 
-		if !f.skipPreflight {
+		if !contains(f.skipPreflight, "true") {
```

Comment:
I'd allow the `*` case as well here; will be more intuitive as we move towards a list of values rather than true/false.